### ISSUE TITLE
Numbing bite could be improved/Balanced maybe?

### ISSUE THREAD

**Issue by Dwaggy** [Repede/YawnWiderStation#182]
So. Custom race trait: Sharp claw/numbing bite.
Specifically the numbing bite part

So..this thought line is a two-different-part idea. but i'll but them in to this one post and split them within it.

### **First off..i feel like the numbing bite needs a slight buff overall**
As it currently stands:
- You bite yourself/your target: doing around 1 to 5 damage in brute.
- You inject roughly 1 to 5 units of stuff (in this case, numbing enzyme) in to them.
- For the Numbing Enzyme specifically, OD is at 20 units (after which things get a little..chaotic for the user)
20 units was around 3 minutes worth of time. (specifically, 300 seconds)
- However, you have to bite around 5, if your -lucky- times to get to that level, of which, congratulations, you've done enough damage to break the bone-break threshold, and have now probably just destroyed your bone on that limb.

I dunno. I mean..i might be biased since im one of a few (that i know of) that uses this trait..and i've always felt it..lacking in any use..

- Combat? Good luck swapping between preferred combat attacks mid combat anyway? but yeah sure this is a concern, though you are injuring yourself and causing bleeding for it.
- Actual painkiller? ...why? at the rates it does, your lucky to give yourself *maybe* 20 seconds worth of painkiller... at the cost of a bite that does damage and causes about 90% bleeding chance, you can literally find painkillers that last a shit load longer, if not a lilbitweaker.

Sure this trait is only 2 (technically one if you wanted the claws anyway), point(s), but it feels..kind of wasted right now. the situations it would be useful in, it either causes so much damage to get a decent amount in, or just..generally doesn't last long enough.

### **Second part to this whole thing**
There's..a lot that could be done with this trait honestly. if there was a way to select chemicals from a list, or choose from different chemicals in some way, swapping between painkillers, to sophoric, to other chemicals that could be seen as part of a system. 
this could be done in 1 of 2 ways:
one: different traits for different bites.
This one would be the easiest way, though it would clutter the perk screen unless it could be selected on another screen. But to make each 'chemical' fang a different perk, selecting the one you wanted that way, giving you the sharp claws, plus the 'chemical name' fang.

Second way would be a little more time consuming perhaps, but be tidier in the perks list.
Turn the fangs in to a similar system to the medical rigs synthesizer/injection system. allowing you to select from a few choice chemicals, synthesize them and store an amount (perhaps at the cost of nutrition), and then allowing you to inject via bite as per normal, or perhaps a different verb that requires a grab of the target (unless its yourself), to inject in to them.

### **third part/after thought.**
So..as of a mention to the last little bit, another thought is *how* the chems are injected. currently its a bite attack move that does damage to the target. injecting a minimal amount of chems based on damage done (i think)
however..this *could* also be changed to something similar to the Vampire drain/succubus drain possibly. In that you'd grab the target, perhaps aggressive, and then use a verb to (with a delay), bite the target for some damage to the targeted limb + bleeding perhaps, and inject an amount in to them that way, and repeating the move until its interupted/stopped. though that might mess with the 'been able to bite yourself' part of it. which..is preferable to still have,

(Sorry if this is a mess to read. its 11:46pm here. and these things came to my head after i had talked it through with like, 2 people that have seen/used the bite too. other than myself. and i wanted to get it down before i forgot.)
and is also a fair bit more work than the other thoughts in this list i'd suspect.

**Comment by Pixelnator** [comment]
I'm all for making it more into an RP tool than a broken and un-used combat tool. And it might be nice to be able to bleed your chems into a beaker too. Obviously making the stuff would drain nutrients (based on the usefulness of the chem). It'd mean removing them completely from the list of attacks (and thus the harm verb) and making them into a verb under the ability tab though.

~~Besides, this way you could totally give people a bite that drugs the target or injects aphrodisiac~~

**Comment by Dwaggy** [comment]
Yep. making them their own verb with time delays would pretty much remove its ability as a combat chem unless you could disengage/get a few seconds to inject yourself/the target with the chem in question. (yes. i maintain that been able to inject yourself is preferable. while it means it can be somewhat abused, it also plays in to roleplay elements too, See a stressed/heavily injured Saur for example.)

Right now its..mostly seen as a combat tool that...also then barely does anything anyway..RP wise its lacking any versatility in what you inject or how much or..anything of the sort, really. and can do potentially large damage after a few bites.

**Comment by izac112** [comment]
If we are going to do any major changes of any trait i think it should be made its own separate new trait with the name indicating its different then the original one in some way.

**Comment by Dwaggy** [comment]
So perhaps one version of the bite that works as is now, (though perhaps injecting *slightly* more in than it does currently, it currently does more damage than it does injection), and then a version that acts as a vampire trait where you grab then and hold and bite. maybe. (though both would have to give the claws and be basically alternate choices of it, i suppose.)
